pycounter/sushi.py

```diff
@@ -248,7 +257,11 @@ def raw_to_full(raw_report):
 
     # Missing some mandatory field to extract data ->
     # exit right away
-    if not c_report or not hasattr(c_report, "Customer") or not hasattr(c_report.Customer, "ReportItems"):
+    if (
```

Comment:
This looks like 'blacking', are you sure we want to reformat external library code? Or is there some change I missed?

Reply:
Actually this change fixes it. I manage to push a commit without a proper 'blacking'.